builder/qemu/config.go

```diff
@@ -758,6 +798,11 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 			errs, fmt.Errorf("net_bridge is only supported in Linux based OSes"))
 	}
 
+	if len(c.BootCommand) > 0 && len(c.BootSteps) > 0 {
+		errs = packersdk.MultiErrorAppend(errs,
+			fmt.Errorf("Both boot_command and boot_steps cannot be used"))
```

Comment:
Come to think of this message, the phrasing might be misunderstood as "you can't use boot_command or boot_steps".
Maybe this could be rephrased to `boot_command and boot_steps cannot be used together`

Reply:
Fixed in d107275f3b617e11134c2201bf0589215b58f202.